master/internal/api_trials.go

```diff
@@ -501,13 +501,16 @@ func (a *apiServer) GetTrial(_ context.Context, req *apiv1.GetTrialRequest) (
 	switch err := a.m.db.QueryProtof(
 		"proto_get_trial_workloads",
 		[]interface{}{
+			"total_batches",
+			db.OrderByToSQL(apiv1.OrderBy_ORDER_BY_ASC),
 			db.OrderByToSQL(apiv1.OrderBy_ORDER_BY_ASC),
 			db.OrderByToSQL(apiv1.OrderBy_ORDER_BY_ASC),
 		},
 		&workloads,
 		req.TrialId,
-		nil,
-		nil,
+		0,
+		1000000,
```

Comment:
so we are assuming trial has <= million batches? What, if anything, goes wrong if this assumption is not correct? 

maybe it's not so important because if you care about workloads you should be using the workloads endpoint?

it looks like we are still getting workloads as a part of trial API. is the plan to get rid of it eventually?

Reply:
I'll replace this with a '0' (i.e. no end batch)
We will later remove workloads response from this endpoint, but the frontend still expects it to show total combined checkpoint size, metric names, whether metrics have been logged, TrialsComparisonModal, and (previously, I think?) the chart information

Reply:
ended up being nil for consistency ✅